src/ucp/core/ucp_ep.c

```diff
@@ -1183,6 +1185,14 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker, const ucp_ep_params_t *params,
     unsigned flags = UCP_PARAM_VALUE(EP, params, flags, FLAGS, 0);
     ucs_status_t status;
 
+    if ((UCP_PARAM_VALUE(EP, params, err_mode, ERR_HANDLING_MODE,
+                         UCP_ERR_HANDLING_MODE_NONE) !=
```

Comment:
need to think of way to enable gva by default w/o disabling error handling, for example when a memh is used for a transaction that requires error handling then re-register it without gva